backend/substrapp/compute_tasks/outputs.py

```diff
@@ -105,6 +104,15 @@ def _save_model_to_local_storage(self, model: context.OutputResource) -> dict:
             },
         }
 
+    @retry(raise_exception=True)
+    def _register_models_in_api(self, registered_models: dict):
+        return organization_client.post(
+            self._ctx.channel_name,
+            settings.LEDGER_MSP_ID,
+            settings.DEFAULT_DOMAIN + reverse("api:model-list"),
+            data={"metadata": json.dumps(registered_models)},
```

Comment:
why wrap behind a metadata key?

Reply:
See this thread : https://stackoverflow.com/questions/43600125/nested-dictionaries-to-json-for-the-post-request-python
We cannot send nested dictionaries with python requests

Reply:
Thanks